SPECS/openssl/openssl.spec

```diff
@@ -1,76 +1,178 @@
-# Don't depend on bash by default
-%define __requires_exclude ^/(bin|usr/bin).*$
-%define soversion 1.1
-Summary:        Utilities from the general purpose cryptography library with TLS implementation
-Name:           openssl
-Version:        1.1.1k
-Release:        27%{?dist}
-License:        OpenSSL
+# For the curious:
```

Comment:
Consider scrubbing the 0.9,8 and 1.0.0 commentary.  Could also scrub the 1.1.0 too